features/Steps/EventSteps.php

```diff
@@ -58,6 +58,19 @@ public function iGetTheEventAt(string $url): void
         $this->theResponseBodyShouldBeValidJson();
     }
 
+    /**
+     * @Given I fail to get the event at :url
+     */
+    public function iFailToGetTheEventAt(string $url): void
```

Comment:
I think the term `FailToGet` is too broad, is it because it does not exist, you do not have permission, it is in the past,...
Since it's a `404` maybe something along the line of `iDoNotFindtheEvent()`

Reply:
I would say there is even no reason to write a new step.

The scenario should be:
- Create a new event and store the UUID
- Get a place with that UUID and check if the error response is 404

Reply:
Maybe a step like `iGetTheEventAtAndTheResponseIs` could help.